bin/i18n/resources/animations.rb

```diff
@@ -0,0 +1,23 @@
+require 'fileutils'
+require 'json'
+
+require_relative '../i18n_script_utils'
+require_relative '../../animation_assets/manifest_builder'
+
+module I18n
+  module Resources
+    module Animations
+      SYNC_IN_RESULT_FILE_PATH = CDO.dir(
```

Comment:
Not blocking, but I think there has been some back and forth with how to name this variable.
This variable stores the full file path where synced-in content will be located inside the `i18n/source` directory to be uploaded to Crowdin. I'd like to see something more simple like `FILE_SOURCE_PATH` since we are going to need a `FILE_BACKUP_PATH` in the redaction of some files.

Reply:
`FILE_SOURCE_PATH` sounds like a place from where we get the data rather than where we place it